lib/views/background_sound/background_sound_view.dart

```diff
@@ -168,4 +161,18 @@ class _BackgroundSoundViewState extends ConsumerState<BackgroundSoundView> {
       ),
     );
   }
+
+  SoundListTileWidget _buildNoneItem() {
+    const noneId = '0';
+    const noneDuration = 0;
+
+    return SoundListTileWidget(
+      sound: BackgroundSoundsModel(
+        id: noneId,
+        title: StringConstants.none,
+        duration: noneDuration,
+        path: '',
+      ),
+    );
+  }
```

Comment:
Sorry I meant defaults of BackgroundSoundsModel.  Do you think that's a good idea?

Reply:
This is good though too, much clearer that is the one for None :)